importer/timestamp_test.go

```diff
@@ -0,0 +1,88 @@
+package importer_test
+
+import (
+	"testing"
+
+	"github.com/projectcypress/cdatools/importer"
+)
+
+func TestTimestampToSeconds(t *testing.T) {
+	tests := []struct {
+		// test case name
+		name string
+		// input to TimestampToSeconds
+		n string
+		// true indicates the return should have been nil
+		expectNil bool
+	}{
+		{
+			"Empty String",
+			"",
+			true,
+		},
+		{
+			"Alphabet Soup",
+			"asdfghjk",
+			true,
+		},
+		{
+			"Proper date without hour, minute, or second",
+			"20160808",
+			false,
+		},
+		{
+			"Proper date with hour",
+			"2016080834",
+			false,
+		},
+		{
+			"Proper date with hour and minute",
+			"201608083434",
+			false,
+		},
+		{
+			"Proper date with hour, minute, and second",
+			"20160808343434",
+			false,
+		},
+		{
+			"Proper date with letters at seconds position",
+			"201608083434as",
+			true,
+		},
+		{
+			"Proper date with letters at hour position",
+			"20160808as3434",
+			true,
+		},
+		{
+			"Proper date with letters at minute position",
+			"2016080834as34",
+			true,
+		},
+		{
+			"Proper string, but length 9 instead of 8",
+			"201608083",
+			true,
+		},
+		{
+			"Proper string, but length 11 instead of 10",
+			"20160808343",
+			true,
+		},
+		{
+			"Proper string, but length 13 instead of 12",
+			"2016080834343",
+			true,
+		},
+	}
+
+	for _, tt := range tests {
+		actual := importer.TimestampToSeconds(tt.n)
+		if actual == nil && !tt.expectNil {
+			t.Errorf("test case: %s: actual was nil instead of producing a timestamp", tt.name)
+		} else if actual != nil && tt.expectNil {
+			t.Errorf("test case: %s: actual was: %d instead of nil\n", tt.name, *actual)
+		}
```

Comment:
When writing this code I was thinking that would be a moving target, but yes now I realize that the result should be the same every time since the date given is static. I can add that in.